### DISCUSSION TITLE
back to "displayShow" from "testRename"

### DISCUSSION THREAD

**Discussion by xbette** [pymedusa/Medusa#9961] [Ideas & Feature requests]
After some earlier updates you made some changed that removed the refresh page thingy

When you pressed the "rename selected" button then the page made a refresh and went back to "displayShow" it does not do that anymore, can you fix that again :D You made a quick fix a week or two ago with the back arrow on the testRename page (works okay) put the original option is better ;D

**Comment by p0psicles** [comment]
Fixed in develop
